remix.init/index.js

```diff
@@ -14,15 +15,18 @@ async function main({ rootDirectory }) {
   const EXAMPLE_ENV_PATH = path.join(rootDirectory, ".env.example");
   const ENV_PATH = path.join(rootDirectory, ".env");
   const PACKAGE_JSON_PATH = path.join(rootDirectory, "package.json");
+  const README_PATH = path.join(rootDirectory, "README.md");
 
   const DIR_NAME = path.basename(rootDirectory);
   const SUFFIX = getRandomString(2);
   const APP_NAME = DIR_NAME + "-" + SUFFIX;
+  const CLOUDFORMATION_NAME = toLogicalID(APP_NAME)
```

Comment:
```suggestion
```

Reply:
hey yep, i was going to use it and then didn't 🙈